packages/eslint-plugin-choreography-ts/tests/no-renaming-operator.test.ts

```diff
@@ -58,5 +58,29 @@ ruleTester.run("no-renaming-operator", noRenameRule, {
         },
       ],
     },
+    {
+      name: "test for error in choreography as an argument",
```

Comment:
Can you also add a test case for `call`?

Reply:
Done